infrastructure/src/main/java/org/corfudb/infrastructure/ManagementAgent.java

```diff
@@ -187,10 +200,15 @@ private void initializationTask() {
             }
         }
 
-        // Start monitoring services that deals with failure and healing detection.
+        // Start management services that deals with failure & healing detection and auto commit.
         if (!shutdown) {
             localMonitoringService.start(METRICS_POLL_INTERVAL);
             remoteMonitoringService.start(POLICY_EXECUTE_INTERVAL);
+            if (!serverContext.getServerConfig(Boolean.class, "--no-auto-commit")) {
```

Comment:
please don't use the double negative, it confuses a lot

Reply:
Because by default auto commit is on, then if user specifies this flag, it's disabled. So has to use double negative here.